torch/csrc/jit/passes/quantization/insert_quant_dequant.cpp

```diff
@@ -430,6 +442,12 @@ class InsertQuantDeQuantHelper {
   std::unordered_map<Graph*, c10::QScheme> qscheme_for_graph_;
 
   bool is_dynamic_ = false;
+
+  // Map from values in original graph to corresponding values in the extracted
+  // subgraph.
+  std::unordered_map<Value*, Value*> remap_values_in_observer_subgraph_;
```

Comment:
can you separate the code that clones a graph to another helper class?

Reply:
Is the main reason to move it this map? I can keep this local to `extractAndRunWeightObserver` and pass it in each time I build the subgraph.

Reply:
that's one concern yes, separate this to a different class also helps with the readability and maintainability